lib/api/src/js/externals/memory.rs

```diff
@@ -183,6 +183,11 @@ impl Memory {
     pub fn is_from_store(&self, _store: &impl AsStoreRef) -> bool {
         true
     }
+
+    pub fn to_shared(&self, _store: &impl AsStoreRef) -> Option<crate::SharedMemoryHandle> {
```

Comment:
probably want to name it `as_shared` since this is not consuming self?

Reply:
Good point, done.